recipe/meta.yaml

```
@@ -21,12 +21,15 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake
     - make  # [unix]
-    - pandoc  # [not ppc64le]
+    - pandoc
   host:
-    - gettext  # [unix]
-    - libogg 1.3.*
+    - gettext 0.21.0 # [unix]
+    - libogg 1.3.5
```

```suggestion
    - libogg 1.3.5
    # libiconv is included in libc for Linux and not used on Windows.
    - libiconv 1.16  # [osx]
```
`libiconv` is needed for osx, you need need to add it to the `host` section too. If we don't do that, our build will depend on a transitive dependency which could break future builds if that dependency is removed in the future.